.ci/install_cc_env_ubuntu.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+echo "Add clear containers sources to apt list"
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/xUbuntu_16.10/ /' >> /etc/apt/sources.list.d/cc-oci-runtime.list"
+
+echo "Install chronic"
+sudo apt-get install -y moreutils
+
+echo "Install rpm2cpio"
+chronic sudo apt-get install -y rpm2cpio
+
+echo "Update apt repositories"
+chronic sudo apt-get update
+
+echo "Install linux-container kernel"
+chronic sudo apt-get install -y --force-yes linux-container
+
+echo "Install qemu-lite binary"
+chronic sudo apt-get install -y --force-yes qemu-lite
+
+echo "Download clear containers image"
+latest_version=$(curl -sL https://download.clearlinux.org/latest)
```

Comment:
The issue with using the latest version is that what we are going to test is changing everytime we test again. That's one of the reasons behind versions.txt in 2.x.